tests/e2e/specs/wcpay/shopper/shopper-checkout-multi-currency.spec.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { shopper, merchant } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+
```

Comment:
Nit: Should we remove this empty line?

Reply:
Removed on https://github.com/Automattic/woocommerce-payments/pull/7804/commits/bf66070dcd8b264b7fbf7a11ff774b046db49387